docker-cli/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9-slim
+
+RUN apt-get update \
+    && dpkg --add-architecture arm64 \
+    && apt-get install -y --no-install-recommends ssh curl python3-dev libffi-dev gcc \
```

Comment:
what's python3-dev?

Reply:
it was needed to install the rumal.yaml.clib because there was no wheel for this arch. I just checked and they added a new one in august so it's redundant. We might be able to remove a some from the runner dockerfile as well